[OOB 5.6] Incomplete Realtek RTL8195AM Instructions

No library included for RTL8195AM board to use with WiFi example.  Instructions mention only two modules (esp8266 and idw01m1).

[REALTEK-RTL8195AM](https://os.mbed.com/platforms/REALTEK-RTL8195AM) page does not give pinout information for wifi pins in mbed_app.json

```
mbed target
[mbed] REALTEK_RTL8195AM
```

```
mbed toolchain
[mbed] arm
```

Compiling results in a error around the ATParser due to no library

```
mbed compile
Building project mbed-os-example-wifi (REALTEK_RTL8195AM, ARM)
Scan: .
Scan: mbed
Scan: env
Scan: FEATURE_LWIP
Compile [ 97.8%]: BufferedSerial.cpp
[Warning] BufferedSerial.cpp@31,0:  #1361-D: function "mbed::SerialBase::attach(T *, void (T::*)(), mbed::SerialBase::IrqType) [with T=BufferedSerial]"  (declared at line 117 of "./mbed-os/drivers/SerialBase.h") was declared "deprecated"
[Warning] BufferedSerial.cpp@156,0:  #1361-D: function "mbed::SerialBase::attach(T *, void (T::*)(), mbed::SerialBase::IrqType) [with T=BufferedSerial]"  (declared at line 117 of "./mbed-os/drivers/SerialBase.h") was declared "deprecated"
Compile [ 98.1%]: ATParser.cpp
Compile [ 98.4%]: ESP8266.cpp
Compile [ 98.8%]: ESP8266Interface.cpp
Compile [ 99.1%]: BufferedSerial.cpp
Compile [ 99.4%]: ATParser.cpp
Compile [ 99.7%]: SPWFSA01.cpp
[Error] SPWFSA01.cpp@26,0:  #289: no instance of constructor "ATParser::ATParser"  matches the argument list
[Error] SPWFSA01.cpp@27,0:  #20: identifier "PC_12" is undefined
[ERROR] "./wifi-x-nucleo-idw01m1/SPWF01SA/SPWFSA01.cpp", line 26: Error:  #289: no instance of constructor "ATParser::ATParser"  matches the argument list
            argument types are: (BufferedSerial, const char [2], const char [2])
"./wifi-x-nucleo-idw01m1/SPWF01SA/SPWFSA01.cpp", line 27: Error:  #20: identifier "PC_12" is undefined
./wifi-x-nucleo-idw01m1/SPWF01SA/SPWFSA01.cpp: 0 warnings, 2 errors

[mbed] ERROR: "/usr/bin/python" returned error code 1.
[mbed] ERROR: Command "/usr/bin/python -u /home/ubuntu/Downloads/mbed-os-example-wifi/mbed-os/tools/make.py -t arm -m REALTEK_RTL8195AM --source . --build ./BUILD/REALTEK_RTL8195AM/arm" in "/home/ubuntu/Downloads/mbed-os-example-wifi"
```

--- a/.mbedignore
+++ b/.mbedignore
@@ -1,0 +1,2 @@
+wifi-x-nucleo-idw01m1/*
+esp8266-driver/*

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ To connect the [X-NUCLEO-IDW01M1](https://developer.mbed.org/components/X-NUCLEO
    Edit ```mbed_app.json``` to include correct WiFi shield, SSID and Password:
 
    ```
-       "config": { 
+       "config": {
  	  "wifi-shield": {
                "help": "Options are WIFI_ESP8266, WIFI_IDW01M1",
                "value": "WIFI_IDW01M1"
@@ -57,10 +57,13 @@ To connect the [X-NUCLEO-IDW01M1](https://developer.mbed.org/components/X-NUCLEO
        },
    ```
 
+   For built-in WiFi, ignore the value of `wifi-shield`
+
 3. Copy the respective _ignore_ file to `.mbedignore`
    * Copy `esp8266-mbedignore` when using ESP8266 module.
    * Copy `idw01m1-mbedignore` when using [X-NUCLEO-IDW01M1](https://developer.mbed.org/components/X-NUCLEO-IDW01M1/) expansion board.
-  
+   * For built-in WiFi, keep the default `.mbedignore`
+
 4. Compile and generate binary
 
    For example, for `GCC`:

--- a/main.cpp
+++ b/main.cpp
@@ -28,16 +28,7 @@ OdinWiFiInterface wifi;
 #include "RTWInterface.h"
 RTWInterface wifi;
 
-#else
-#if MBED_CONF_APP_WIFI_SHIELD == WIFI_ESP8266
-#if !TARGET_FF_MORPHO
-#error [NOT_SUPPORTED] Only Morpho form factor devices are supported for this shield at this time
-#endif // !TARGET_FF_MORPHO
-#else
-#if !TARGET_FF_ARDUINO
-#error [NOT_SUPPORTED] Only Arduino form factor devices are supported for this shield at this time
-#endif // !TARGET_FF_ARDUINO
-#endif // MBED_CONF_APP_WIFI_SHIELD != WIFI_ESP8266
+#else // External WiFi modules
 
 #if MBED_CONF_APP_WIFI_SHIELD == WIFI_ESP8266
 #include "ESP8266Interface.h"
@@ -47,7 +38,7 @@ ESP8266Interface wifi(MBED_CONF_APP_WIFI_TX, MBED_CONF_APP_WIFI_RX);
 SpwfSAInterface wifi(MBED_CONF_APP_WIFI_TX, MBED_CONF_APP_WIFI_RX);
 #endif // MBED_CONF_APP_WIFI_SHIELD == WIFI_IDW01M1
 
-#endif // TARGET_UBLOX_EVK_ODIN_W2
+#endif
 
 const char *sec2str(nsapi_security_t sec)
 {

